### PR TITLE
Sils/langs

### PR DESCRIPTION
**Pull request by sils1297**

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
ack 42f70e8

File: tests/output/ConsoleInteractionTest.py [hunk 1]
```diff
@@ -613,6 +618,8 @@ def test_print_bears_no_settings(self):
             print_bears(self.log_printer.printer, bears, False)
             expected_string = "SomeBear:\n"
             expected_string += "  Some Description.\n\n"
+            expected_string += ("  The bear does not provide information "
```
Comment (Adrianzatreanu) [inline tests/output/ConsoleInteractionTest.py]: why exactly are we leaving 2 spaces before?

Comment (sils1297) [inline tests/output/ConsoleInteractionTest.py]: the whole bear information is indented as all the other paragraphs since it's a long list of bears :)

**Comment by Adrianzatreanu** [comment]
ack 19d750a

**Comment by sils1297** [comment]
ack 19d750a ?

**Comment by sils1297** [comment]
@rultor merge

**Comment by sils1297** [comment]
@rultor merge

**Comment by sils1297** [comment]
@rultor merge

**Comment by sils1297** [comment]
@rultor merge

**Comment by sils1297** [comment]
@rultor merge

**Comment by Adrianzatreanu** [comment]
@rultor merge , there's no other option, OBEY ME

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**